packages/shared/src/domain/address/state.ts

```diff
@@ -16,6 +16,64 @@ export function normalizeState(state: string): USState {
 }
 
 export enum USState {
+  AL = "AL",
+  AK = "AK",
+  AS = "AS",
+  AZ = "AZ",
+  AR = "AR",
+  CA = "CA",
+  CO = "CO",
+  CT = "CT",
+  DE = "DE",
+  DC = "DC",
+  FL = "FL",
+  GA = "GA",
+  GU = "GU",
+  HI = "HI",
+  ID = "ID",
+  IL = "IL",
+  IN = "IN",
+  IA = "IA",
+  KS = "KS",
+  KY = "KY",
+  LA = "LA",
+  ME = "ME",
+  MD = "MD",
+  MA = "MA",
+  MI = "MI",
+  MN = "MN",
+  MS = "MS",
+  MO = "MO",
+  MT = "MT",
+  NE = "NE",
+  NV = "NV",
+  NH = "NH",
+  NJ = "NJ",
+  NM = "NM",
+  NY = "NY",
+  NC = "NC",
+  ND = "ND",
+  OH = "OH",
+  OK = "OK",
+  OR = "OR",
+  PA = "PA",
+  PR = "PR",
+  RI = "RI",
+  SC = "SC",
+  SD = "SD",
+  TN = "TN",
+  TX = "TX",
+  UT = "UT",
+  VT = "VT",
+  VA = "VA",
+  VI = "VI",
+  WA = "WA",
+  WV = "WV",
+  WI = "WI",
+  WY = "WY",
+}
+
+export enum USStateWithoutTerritories {
```

Comment:
Because of your previous comment @leite08 I couldnt set the system to n/a so i created another enum that doesnt have territories. These will be used everywhere we set a drivers license since we dont have valid values there.